packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -759,6 +760,7 @@ const defaultConfigs = {
   OCEANUSD: {
     type: "medianizer",
     minTimeBetweenUpdates: 60,
+    historicalTimestampBuffer: 60,
```

Comment:
Should we add to other configs? or more slowly roll this out.

Reply:
I would want to apply it to all financial products at once. however is this _enough_ of a buffer for ocean? it's a bit difficult when there is no volume at all in some of these pairs

Reply:
yeah, perhaps instead of updating every single config, we set the default buffer to 60, and then set ocean's to even wider since we've had problems with it specifically?